### PR TITLE
Main

### PR DESCRIPTION
**Pull request by gdimit01**
Georgi Dimitrov gd353 request pull before I can push a few changes into main please. Thank you

### CODE REVIEW

**Comment by SergeyOvchinnik** [comment]
The change in method x makes sense, we can accept that.

The commit contains some unnecessary changes though. There is a change to the .project file which we don't need and there is a strange set of changes to locateTopEven.

The changes to locateTopEven fix some of the code quality issues but not all of them. The code is still all aligned to the same level, inconsistent with the alignment used in the rest of the file. Additionally, you seem to have added a lot of largely useless comments.

Your comments use "we" a lot to describe the functionality. "We" don't do anything here, the method does, so please avoid using "we" in the comments. Also, you seem to have placed the comments that describe specific lines of code before the method. These comments are usually placed in-line, next to the line they are describing.

Additionally, good comments should add new meaning to the code by describing the functionality in a more understandable way. Some of your comments are largely useless. Comments like `We first declare a variable topEven and initialize it to the smallest possible integer value.`, `We then declare a variable indexOfTopEven and initialize it to 0.`, and `We return indexOfTopEven.` just describe the code itself, adding any new meaning.

In conclusion, please remove the change in the .project file and make improvements to the alignment and comments of the locateTopEven method.
